mig-scheduler/agents_management.go

```diff
@@ -82,8 +82,18 @@ func killDupAgents(queueLoc string, ctx Context) (err error) {
 			// throttling to prevent issuing too many kill orders at the same time
 			time.Sleep(5 * time.Second)
 		} else {
-			desc := fmt.Sprintf("found '%d' agents running on '%s'. Require "+
-				"manual inspection.", remainingAgents, queueLoc)
+			// Build a list of relevant agent names to include in the manual inspection
+			// notification
+			var namelist string
+			for _, agent := range agents {
+				if namelist == "" {
+					namelist = agent.Name
+				} else {
+					namelist += ", " + agent.Name
+				}
+			}
+			desc := fmt.Sprintf("found %v agents running on %v (%v). Require "+
```

Comment:
I'd put the namelist at the very end, not in the middle of the error msg, for readability.